tomb-common/Cargo.toml

```diff
@@ -50,5 +50,8 @@ hex-literal = "^0.4"
 crate-type = ["cdylib", "rlib"]
 path = "src/lib.rs"
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+tokio = { workspace = true }
+
```

Comment:
We don't want to be using workspace specified dependencies, it's tech-debt that we already have so many. The lockfile is already shared so as long as the major semver is the same we get the useful portion of this feature automatically without tightly binding together the workspace deps in hard to untangle ways.